### ISSUE TITLE
Arguments in argparse nonexistent in code

### ISSUE THREAD

**Issue by soniamartinot** [boschresearch/Hyp-OW#2]
Hi, I'm trying to reproduce your results. However I cannot find the use / definition of some of your arguments in the actual code.
For example:
-  the argument "unknown_weight" does not appear anywhere in the code,
- "compare_family" (in main) is not defined anywhere,
- "distributed" is not defined anywhere,

If some issues like missing "distributed" definition can be quite quickly solved, the defined arguments that don't exist in the code are puzzling.
Could you please provide some insights on this or an up-to-date code ?

Also, how did you set all your hyperparameters ? Did you use a validation set ?

Thank you in advance

**Comment by tldoan** [comment]
Hi,

After digging a bit deeper (sorry, it's been a while):

- `unknown_weights` doesn't seem to be used—I probably forgot to remove it.
- `compare_family` was used only for ablation experiments, so you can safely discard it.
- `distributed` is necessary only when training on multiple GPUs.
For the hyperparameters, I followed the experimental settings described in [PROB](https://arxiv.org/abs/2212.01424) and [OW-DETR](https://arxiv.org/abs/2112.01513).

Hope it helps!

Thang,
